src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/DefendantResponseCopyContentProvider.java

```diff
@@ -14,27 +15,36 @@
 @Component
 public class DefendantResponseCopyContentProvider {
 
-    private final DefendantDetailsContentProvider defendantDetailsContentProvider;
+    private final PartyDetailsContentProvider partyDetailsContentProvider;
+    private final ClaimContentProvider claimContentProvider;
 
-    public DefendantResponseCopyContentProvider(final DefendantDetailsContentProvider defendantDetailsContentProvider) {
-        this.defendantDetailsContentProvider = defendantDetailsContentProvider;
+    public DefendantResponseCopyContentProvider(
+        final PartyDetailsContentProvider partyDetailsContentProvider,
+        final ClaimContentProvider claimContentProvider
+    ) {
+        this.partyDetailsContentProvider = partyDetailsContentProvider;
+        this.claimContentProvider = claimContentProvider;
     }
 
     public Map<String, Object> createContent(final Claim claim) {
         requireNonNull(claim);
         ResponseData defendantResponse = claim.getResponse().orElseThrow(IllegalStateException::new);
 
         Map<String, Object> content = new HashMap<>();
-        content.put("claimReferenceNumber", claim.getReferenceNumber());
-        content.put("claimSubmittedOn", formatDate(claim.getCreatedAt()));
-        content.put("claimantType", PartyUtils.getType(claim.getClaimData().getClaimant()));
-        content.put("claimantFullName", claim.getClaimData().getClaimant().getName());
-        content.put("defendant", defendantDetailsContentProvider.createContent(
+
+        content.put("defendant", partyDetailsContentProvider.createContent(
+            claim.getClaimData().getDefendant(),
+            defendantResponse.getDefendant(),
+            claim.getDefendantEmail()
+        ));
+        content.put("claimant", partyDetailsContentProvider.createContent(
             claim.getClaimData().getDefendant(),
-            defendantResponse,
+            claim.getClaimData().getClaimant(),
             claim.getDefendantEmail()
         ));
+        content.put("claimantType", PartyUtils.getType(claim.getClaimData().getClaimant()));
```

Comment:
That is a duplicate, a `claimant.type` is provided by `partyDetailsContentProvider`.